lib/std/base64.zig

```diff
@@ -301,6 +301,33 @@ pub const Base64Decoder = struct {
             if (padding_chars != padding_len) return error.InvalidPadding;
         }
     }
+
+    // destWriter must be compatible with std.io.Writer's writeAll interface
```

Comment:
This (and all comments below) ought to be doc comments - `///`, proper code formatting and punctuation:

```suggestion
    /// `dest_writer` must be compatible with `std.io.Writer`'s `writeAll` interface.
```

Also camel case -> snake case as per https://ziglang.org/documentation/master/#Names

Reply:
Thank you. This should be fixed in https://github.com/ziglang/zig/pull/21348/commits/29b004d16cc09d97aafdfc340c5281c3783bdea2 .

I took the opportunity to fix it too in the decoding to stream part, as it was my code originally and had the same mistakes.